### DISCUSSION TITLE
Visual Studio 2022 freezes when using copilot

### DISCUSSION THREAD

**Discussion by rasputino** [community/community#13933] [Copilot Conversations] [answered]
I have solution with VB.Net and C# mixed projects. When Visual studio 2022 starts it freezes and I have to close it in task manager.

**Comment by kzu** [comment, marked as answer]
Could you try updating to the latest version (v1.10.0.2)? It improves a number of scenarios and should also fix this issue.

Thanks!

**Reply by rasputino** [reply]
Updated and fixed!

**Comment by victorlmtavares** [comment]
Had the same here, updating to 1.10.0.2 fixed it. Thank you guys!
